frontend/src/styles/components/_DialogModule.sass

```diff
@@ -5,4 +5,4 @@
     text-transform: capitalize
 
   [class^="Table-module_actionContainer"]
-    display: none
+    display: none  
```

Comment:
Two random spaces here 😅

Reply:
Haha - it seems that's what you get when you try out a different editor. Tested VSCode, and it didn't have a eslint/prettier fix-on-save enabled. :D